*** AppController_orig.mm	Fri Mar 22 03:13:55 2013
--- AppController.mm	Fri May 17 05:33:56 2013
***************
*** 13,18 ****
--- 13,19 ----
  #import <OpenGLES/ES2/glext.h>
  
  #include <mach/mach_time.h>
+ #import <GooglePlus/GooglePlus.h>
  
  // USE_DISPLAY_LINK_IF_AVAILABLE
  //
*************** void NotifyFramerateChange(int targetFPS
*** 806,811 ****
--- 807,827 ----
      ReleaseViewHierarchy();
      [super dealloc];
  }
+ 
+ - (BOOL)application:(UIApplication *)application
+             openURL:(NSURL *)url
+   sourceApplication:(NSString *)sourceApplication
+          annotation:(id)annotation {
+   
+   if([sourceApplication isEqualToString:@"com.google.chrome.ios"] ) {
+     NSLog(@"GooglePlus URL %@, app %@, sourcApp%@", url,sourceApplication, application);
+     return [GPPURLHandler handleURL:url
+                   sourceApplication:sourceApplication
+                          annotation:annotation];
+   }
+   return NO;
+ }
+ 
  @end
  
  
